adding Postman collection of sample UMAPI calls  - fixes #58

Luci2015 opened the pull request.

adorton-adobe commented:
This is fantastic and I think we should plan to get it merged.

One question - will this work with the Postman environment you can download from the developer console? When I installed your collection I couldn't figure out a way to merge them automatically. I just copied and pasted the stuff from the environment.

Luci2015 commented:
I do not know if you can merge between global environment variables and local collection variables. 
Thing is, I embedded the "obtain access token" call from Developer Portal, but because the pre-request script was referencing the global variables, I modified the script to look for local collection variables  instead (pm.collectionVariables vs pm.environment) so that I could export the collection with collection variables. Also, I added the UMAPI_ prefix as well, just to be more clear on the purpose it is used for.

adorton-adobe commented on docs/en/samples/UMAPI SAMPLES.postman_collection.json:
```diff
@@ -0,0 +1,1543 @@
+							"        \"aud\": \"https://\" + pm.collectionVariables.get(\"UMAPI_IMS\") + \"/c/\" + pm.collectionVariables.get(\"UMAPI_API_KEY\")",
+							"    };",
+							"    ",
+							"    var meta_scope = \"https://ims-na1.adobelogin.com/s/ent_user_sdk\";",
```
This won't work when `UMAPI_IMS` is set to the stage IMS host. We should build the metascope URI with that variable:

```
var meta_scope = "https://"+pm.collectionVariables.get("UMAPI_IMS")+"/s/ent_user_sdk";
```

adorton-adobe commented on docs/en/samples/UMAPI SAMPLES.postman_collection.json:
```diff
@@ -0,0 +1,1543 @@
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://usermanagement.adobe.io/v2/usermanagement/users/{{UMAPI_IMS_ORG}}/{{UMAPI_PAGE}}",
```
We should govern the API host with a variable so we can easily target stage. Otherwise there are 23 instances of the hostname in the file that one would need to manually override.

rwunsch commented:
This is great! I was looking right now for exactly that collection - and initially did not find it.
Some Slack messages later, I was here.
Definitely a commit to be added!!

lesavage commented on docs/en/samples/index.md:
```diff
@@ -208,4 +210,19 @@ def make_call(self, path, body=None):
+
+ Download [this](UMAPI_SAMPLES.postman_collection.json) json file locally and import it to your Postman environment.  
+ Click Collections menu in Postman and select the `UMAPI SAMPLES` to view its Variables menu in the main page area.  
+ Fill in the `CURRENT VALUE` field the associated values obtained from your [developer portal](https://developer.adobe.com)'s integration/project (at least the first 5 rows)  
```
```suggestion
 Fill in the `CURRENT VALUE` field with the associated values obtained from your [developer portal](https://developer.adobe.com)'s integration/project (at least the first 5 rows)  
```

lesavage commented on docs/en/samples/index.md:
```diff
@@ -208,4 +210,19 @@ def make_call(self, path, body=None):
+ Download [this](UMAPI_SAMPLES.postman_collection.json) json file locally and import it to your Postman environment.  
+ Click Collections menu in Postman and select the `UMAPI SAMPLES` to view its Variables menu in the main page area.  
+ Fill in the `CURRENT VALUE` field the associated values obtained from your [developer portal](https://developer.adobe.com)'s integration/project (at least the first 5 rows)  
+ Before running any API in the list, you need to run first the `Auth - Obtain Access Token` to have an active token for this session.  
```
```suggestion
 Before running any API in the list, you need to run `Auth - Obtain Access Token` first to have an active token generated for the session.  
```

lesavage commented on docs/en/samples/index.md:
```diff
@@ -208,4 +210,19 @@ def make_call(self, path, body=None):
+ Fill in the `CURRENT VALUE` field the associated values obtained from your [developer portal](https://developer.adobe.com)'s integration/project (at least the first 5 rows)  
+ Before running any API in the list, you need to run first the `Auth - Obtain Access Token` to have an active token for this session.  
+ 
+ 
+ 
+ 
```
```suggestion
```

lesavage reviewed (commented):
Some comments on documentation.

rwunsch commented:
Thanks @lesavage ! That is super helpful! :)
